packages/dataviews/src/dataforms-layouts/panel/index.tsx

```diff
@@ -111,6 +113,10 @@ function PanelDropdown< Item >( {
 		[ popoverAnchor ]
 	);
 
+	const fieldValue = fieldDefinition.getValue( { item: data } );
+	const showMixedValue =
+		isBulkEditing && ( fieldValue === undefined || fieldValue === '' );
```

Comment:
This is where having a central intersected object falls short, and why I am leaning towards moving the array of items down so the field layout has more control and also optionally the field.
As in this case the `getValue` for title, returns an empty string if it isn't defined. While empty string may be a valid case for another field and different from `undefined` or `null`.

Reply:
@youknowriad  Where was it that you suggested Symbol?
And are you thinking we did use it to define the `Mixed` value. That in the `getIntersectingValues` function for example when it is a mixed value we set it to `Symbol.for( 'unique dataform mixed value key' )`? and than check for that in the above? or are you thinking something else?

I did try the above, but ran into this error `Cannot use 'in' operator to search for 'rendered' in Symbol(DATAFORM_MIXED_VALUE)` within the `getItemTitle` function. So in order to make this work I did probably have to do what I suggested above:
> I am leaning towards pushing the intersecting logic down to the field view and rely on getValue, but this will cascade into quite a few other changes.

Reply:
>  Where was it that you suggested Symbol?

https://github.com/WordPress/gutenberg/pull/67344#discussion_r1863295292


> That in the getIntersectingValues function for example when it is a mixed value we set it to Symbol.for( 'unique dataform mixed value key' )? and than check for that in the above? or are you thinking something else?

yes

---

I'm not sure we should push the merging down to the "field". We should just check where `getItemTitle` is called and not call it when it's a symbol (I'm assuming it's an internal call to data form component)

Reply:
I just saw that some of my comments were not in the right place.

Reply:
> We should just check where getItemTitle is called and not call it when it's a symbol (I'm assuming it's an internal call to data form component)

That would work to, its being called within the `getValue` function of the `title` field definition.